src/ui/src/builder/composables/useDragToSelect.ts

```diff
@@ -0,0 +1,341 @@
+import { ref, computed, type Ref, onUnmounted } from "vue";
+import type { BuilderManagerMode } from "../builderManager";
+import { generateBuilderManager } from "../builderManager";
+
+type BuilderManager = ReturnType<typeof generateBuilderManager>;
```

Comment:
You can use this type.
```suggestion
import type { BuilderManager } from "@/writerTypes";
```